dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java

```diff
@@ -16,17 +16,64 @@
  */
 package com.google.edwmigration.dumper.application.dumper.connector.oracle;
 
+import com.google.auto.value.AutoValue;
 import com.google.common.collect.ImmutableList;
+import com.google.common.io.Resources;
 import com.google.edwmigration.dumper.application.dumper.ConnectorArguments;
+import com.google.edwmigration.dumper.application.dumper.task.JdbcSelectTask;
 import com.google.edwmigration.dumper.application.dumper.task.Task;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.charset.Charset;
 import javax.annotation.Nonnull;
 import javax.annotation.ParametersAreNonnullByDefault;
 
 @ParametersAreNonnullByDefault
 class StatsTaskListGenerator {
 
   @Nonnull
-  ImmutableList<Task<?>> createTasks(ConnectorArguments arguments) {
-    throw new UnsupportedOperationException();
+  ImmutableList<Task<?>> createTasks(ConnectorArguments arguments) throws IOException {
+    ImmutableList.Builder<Task<?>> builder = ImmutableList.<Task<?>>builder();
+
+    ImmutableList<StatsQuery> queries =
```

Comment:
I would suggest extracting this list to a constant.

Reply:
Done